File: app/src/main/java/com/brainwellnessspa/ReferralModule/Model/CheckReferCodeModel.java
package com.brainwellnessspa.ReferralModule.Model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CheckReferCodeModel {
    @SerializedName("ResponseData")
    @Expose
    private ResponseData responseData;
    @SerializedName("ResponseCode")
    @Expose
    private String responseCode;
    @SerializedName("ResponseMessage")
    @Expose
    private String responseMessage;
    @SerializedName("ResponseStatus")
    @Expose
    private String responseStatus;

    public ResponseData getResponseData() {
        return responseData;
    }

    public void setResponseData(ResponseData responseData) {
        this.responseData = responseData;
    }

    public String getResponseCode() {
        return responseCode;
    }

    public void setResponseCode(String responseCode) {
        this.responseCode = responseCode;
    }

    public String getResponseMessage() {
        return responseMessage;
    }

    public void setResponseMessage(String responseMessage) {
        this.responseMessage = responseMessage;
    }

    public String getResponseStatus() {
        return responseStatus;
    }

    public void setResponseStatus(String responseStatus) {
        this.responseStatus = responseStatus;
    }

    public class ResponseData {
        @SerializedName("CodeExist")
        @Expose
        private String codeExist;

        public String getCodeExist() {
            return codeExist;
        }

        public void setCodeExist(String codeExist) {
            this.codeExist = codeExist;
        }
    }
}

File: app/src/main/java/com/brainwellnessspa/referralModule/models/CheckReferCodeModel.kt
package com.brainwellnessspa.referralModule.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class CheckReferCodeModel {
    @SerializedName("ResponseData")
    @Expose
    var responseData: ResponseData? = null

    @SerializedName("ResponseCode")
    @Expose
    var responseCode: String? = null

    @SerializedName("ResponseMessage")
    @Expose
    var responseMessage: String? = null

    @SerializedName("ResponseStatus")
    @Expose
    var responseStatus: String? = null

    inner class ResponseData {
        @SerializedName("CodeExist")
        @Expose
        var codeExist: String? = null
    }
}